src/Plugin.php

```diff
@@ -476,6 +477,17 @@ public static function loadLang($plugin_key, $forcelang = '', $coretrytoload = '
             }
         }
 
+        if (!defined('TU_USER')) {
+            $i18n_cache = new I18nCache((new CacheManager())->getTranslationsCacheInstance());
+            $TRANSLATE = new class ($i18n_cache) extends Laminas\I18n\Translator\Translator {
+                public function __construct(?I18nCache $cache)
+                {
+                    $this->cache = $cache;
+                }
+            };
+            $TRANSLATE->setLocale($coretrytoload);
+        }
+
```

Comment:
If the listener change is effective, this can be removed.

Reply:
Without this code, the initial error comes back.

Reply:
It works on my side, but only after deleting the cache.

Reply:
Yes for me too by emptying the folder `_cache`